fem/bilininteg_ext.cpp

```diff
@@ -753,10 +1018,16 @@ void MassIntegrator::Assemble(const FiniteElementSpace &fes)
    ne = fes.GetMesh()->GetNE();
    nq = ir->GetNPoints();
    dofs1D = el.GetOrder() + 1;
-   quad1D = IntRules.Get(Geometry::SEGMENT, ir->GetOrder()).GetNPoints();
-   geom = GeometryExtension::Get(fes,*ir);
-   maps = DofToQuad::Get(fes, fes, *ir);
-   vec.SetSize(ne*nq);
+
+  static bool Mass_SetSize = true;
+  if(Mass_SetSize) {
+    quad1D = IntRules.Get(Geometry::SEGMENT, ir->GetOrder()).GetNPoints();
+    geom = GeometryExtension::Get(fes,*ir);
+    maps = DofToQuad::Get(fes, fes, *ir);
+    vec.SetSize(ne*nq);
+    Mass_SetSize = false;
+  }
```

Comment:
If we have to reassemble operators at every right hand side evaluation it might be good to fetch this data once, would this be fair? Similar case for other operators.